src/components/results/result.vue

```diff
@@ -23,11 +23,13 @@
         <h1 class="x-small x-text--bold">{{ result.season }}</h1>
         <h2 class="x-text x-text--secondary x-ellipsis">{{ result.name }}</h2>
       </div>
-      <BaseResultCurrentPrice
-        format="i,iii.dd €"
-        :result="result"
-        class="x-text x-text--accent x-text--bold"
-      />
+
+      <!--
```

Comment:
Why are we adding a slot here? This is the archetype and we should not add slots right? Like the name or the season in this component.

Reply:
I added it because one of the requirements of the story is to be able to override the currency in a component prop as CDL has if I'm not wrong.

There were no possibility to do it without this "custom" `Result` slot which is used in many different places like the Recommendations, the ResultsList, etc.

Now you can pass a currency component with the `:format` to do it.

Reply:
It has to be possible, but there is no need to do this here. If it is necessary for some setup/customer it will be done there.